pages/firststeps.md

```diff
@@ -1,12 +1,29 @@
+If you are deemed fit for the internship after completing the steps, you will be officially invited to join the OLE interns team! We’ll add you to the interns Gitter chatroom and assign you to a specific team to work on developing and improving OLE’s BeLL (Basic e-Learning Library) software. The BeLL is a virtual library that is deployed internationally to children in countries that typically do not have access to educational resources.
+
+Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will be working with software and languages including Git, GitHub, Gitter, Markdown, Vagrant, VirtualBox, Command Line/Terminal, Command Line/Terminal Scripts, Vim, CouchDB, Docker, HTML5, Javascript, and Node.js.
+
+**Note:** *This is an unpaid but intensive internship that requires 16 hours of work each week. More information about the internship can be found [here](https://www.indeed.com/jobs?q=%22virtual+software+engineer+intern%22&l=Cambridge,+MA). If you have further questions, don’t hesitate to ask other members in the Gitter [chat](http://open-learning-exchange.github.io/#!pages/chat.md)!*
+
+# The Steps
+
+Social coding is a huge part of any open source and collaborative project, and the Open Learning Exchange is no different. In the following series of steps, you will learn about the Github team workflow, including git, forking, pull requests, and repository issues. You will also be introduced to Markdown’s rich text formatting and OLE’s Basic e-Learning Library (BeLL), the digital library that hosts the learning materials.
+
+Because these steps are simple, we expect high quality work, which may take a longer time. We want to see that you are capable of using or learning how to use these tools (writing good GitHub issues, creating pull requests, navigating the BeLL, etc.). These steps may seem easy, but we want you to impress us with good GitHub etiquette and quality Markdown. The bare minimum would be to just passively follow the steps; you should do further reading about the tools/languages we use so you can further your understanding and relieve confusion if you're unclear about how something works. **Treat these steps as learning opportunities!** The Github and Markdown skills you practice here are very important in both this internship and a future software development career.
+
+We created a [FAQ page](http://open-learning-exchange.github.io/#!pages/faq.md#Helpful_Links) where you can find the answers to some commonly asked questions, both technical and general, and also some very useful links and video tutorials that will help you become familiar with the tools/languages we use. For anything that is not in the FAQ page, Google and Stack Exchange are your friends :)
```

Comment:
@waryhermit I think the paragraph at line 19 should say this:

Each step has useful links at the bottom of the page. Make sure to check these links out. We created a [FAQ page](http://open-learning-exchange.github.io/#!pages/faq.md#Helpful_Links) where you can find the answers to some commonly asked questions, both technical and general, and also some even more useful links and video tutorials that will help you become familiar with the tools/languages we use. For anything that is not in the FAQ page, Google and Stack Exchange are your friends :)